docs/rfcs/2025-04-14-signing-messages.md

```diff
@@ -0,0 +1,83 @@
+# Signing some member member messages and profiles
+
+## Problems
+
+1. Authenticity of profile addresses.
+
+When somebody sends a profile, e.g. when connecting via one time link, it may include contact address. It may be incorrectly used to claim that the sending person is the same person as one who published the same address in some trustworthy channel, like a social media account or a website. This claim is unproven, and can potentially be misleading - the address can be simply copy-pasted, together with other attributes of the profile. The solution to that would be to sign the profile or a message including the profile with the key that has its public part included in immutable address data. If only profile is signed, it can also be copy-pasted, but if the message is signed and if it includes some connection attributes (e.g. security code), then it would be impossible to copy paste. So the latter approach is preferred.
```

Comment:
What does security code in scope of address mean, isn't it some connection encryption derivative?
Also why can't it be copied?

Reply:
it won't match in another conversation. Security code - the same as we show in the app, from double ratchet.